script/post-install.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/sh
 
-rm /usr/bin/pulsar
-cp /opt/Pulsar/resources/pulsar.sh /usr/bin/pulsar
+FILESOURCE='/opt/Pulsar/resources/pulsar.sh'
+FILEDEST='/usr/bin/pulsar'
+
+if [ -f "$FILEDEST" ]
+then
+  rm "$FILEDEST"
+else
+  echo "File '$FILEDEST' not found (this is not an error)"
```

Comment:
Judging from past experience, I think we're still going to get asked about this even though it says it's not an error. :/ What can you do, people are curious.

So I think maybe change the message to be even friendlier/gentler, OR don't log if the file doesn't exist, since we don't really mind if this is a user's first install.

```suggestion
else
  echo "Info: file '$FILEDEST' not found (this is okay and not an error)"
```

Or just 

```suggestion
```

My two cents.

Reply:
Other than that, looks good to me!